src/main/resources/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMNavigator/config.jelly

```diff
@@ -19,6 +19,9 @@
   <f:entry title="${%Owner}" field="repoOwner">
     <f:textbox/>
   </f:entry>
+  <f:entry title="${%Project Key}" field="projectKey">
```

Comment:
you write in '[help-projectKey.html](https://github.com/jenkinsci/bitbucket-branch-source-plugin/pull/654/files#diff-3496a39845edf5848173e4bbedbe887743dd4709e7e8d1157d809ffd2c302b12)' that thsi shall be optional, but here it is mandatory entry

Reply:
Entry is present but there are no checks for it content. It can be left empty and it this sense it is optional.

Reply:
Hi, as I tell you in gitter, I am not owner therefore I am no top permitted to approve this PR.
+ check who was last committers and try to ping them
+ or adopt this plugin. In that case your are the owner and you can merge your PRs without reviews.